### ISSUE TITLE
Elbow instability

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10795]
**Preferred term label:**
Elbow instability
And 2 subterms:
Valgus instability of the elbow
Varus instability of the elbow

**Synonyms**

**Definition (free text, please give PubMed ID)**
Subjective or objective feeling of clunking/clicking in the elbow with movement, elbow 'giving way' or sensation of elbow dislocating

Varus: Elbow varus stress test; lateral collateral ligament injury of the elbow

Valgus: Elbow valgus stress test; medial collateral ligament injury of the elbow

**Parent term (use hpo.jax.org/app)**
sibling term is https://hpo.jax.org/browse/term/HP:0003042, but that is radiological and this is clinical with some overlap with dislocation.

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For varus: Capitellum fracture
For valgus and varus: Radial head fracture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I would propose modifying the definition a bit

Impairment of the ability of the elbow joint to maintain proper anatomical alignment and functional mechanics. Elbow instability is a spectrum from subluxation to dislocation, with corresponding clinical and pathologic features and therapeutic implications. 
PMID: [28160902](https://pubmed.ncbi.nlm.nih.gov/28160902/)
https://pubmed.ncbi.nlm.nih.gov/7962147/

Adding new term: 
- ID: HP:6001426
- label: Elbow instability
- parents: [Abnormality of the elbow (HP:0009811)](https://hpo.jax.org/app/browse/term/HP:0009811)
- pmid: PMID:28160902|PMID:7962147
- def: Impairment of the ability of the elbow joint to maintain proper anatomical alignment and functional mechanics. Elbow instability is a spectrum from subluxation to dislocation, with corresponding clinical and pathologic features and therapeutic implications.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10795
- synonyms:

**Comment by pnrobinson** [comment]
I am going to have to rearrange the hierarchy of the elbow section a bit, but the following two terms will wind up being children of Elbow instability

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001427
- label: Valgus instability of the elbow
- parents: [Abnormality of the elbow (HP:0009811)](https://hpo.jax.org/app/browse/term/HP:0009811)
- pmid: PMID:28160902
- def: A type of elbow instability that arises through injuries to the medial collateral ligament (MCL), specifically the anterior bundle (AMCL). In the valgus stress test, the elbow is placed in 20 to 30 degrees of flexion with the forearm supinated and a valgus stress is applied. The test is positive when no firm end point is palpated, greater than 1 mm of medial joint opening is noted (fluoroscopically), or there is reproduction of the patient's pain.
- comment: Medial collateral ligament injuries may occur after trauma, such as an elbow dislocation, or as a repetitive overuse injury, commonly seen in overhead-throwing athletes. The typical history includes a pop and medial elbow pain following throwing activities, or patients may present with recurrent medial elbow pain, classically during the late cocking to early acceleration phase of throwing. Athletes typically complain of a progressive loss of velocity, accuracy, and endurance, which may be exacerbated by glenohumeral internal rotation deficits common to throwing athletes. The physical examination typically elicits tenderness over the MCL.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10795
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001428
- label: Varus instability of the elbow
- parents: [Abnormality of the elbow (HP:0009811)](https://hpo.jax.org/app/browse/term/HP:0009811)
- pmid: PMID:28160902
- def: A type of elbow insttability characterized by abnormal widening or gapping of the lateral (radial) side of the elbow joint when a varus stress is applied, due to insufficiency of the lateral collateral ligament complex.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10795
- synonyms:
